dandiapi/api/migrations/0006_default_oauth_application.py

```diff
@@ -21,7 +21,7 @@ def create_application(apps, schema_editor):
             client_secret='',
             name=name,
             # This can be turned off in production if appropriate.
-            skip_authorization=False,
+            skip_authorization=True,
```

Comment:
Should this still be modified here? I thought the point of modifying the `StagingApplication` class was to avoid this being changed here directly

Reply:
Nevermind, this was explained to me. This is the normal (non-staging) application, whereas the other migration is for the staging application.